MonoGame.Framework/Graphics/GraphicsDebug.DirectX.cs

```diff
@@ -0,0 +1,82 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using SharpDX.Direct3D11;
+using System.Collections.Generic;
+
+namespace Microsoft.Xna.Framework.Graphics
+{
+    public partial class GraphicsDebug
+    {
+        private readonly GraphicsDevice _device;
+        private readonly InfoQueue _infoQueue;
+        private readonly Queue<GraphicsDebugMessage> _cachedMessages;
+        private bool _hasPushedFilters = false;
+
+        public GraphicsDebug(GraphicsDevice device)
+        {
+            _device = device;
+            _infoQueue = _device._d3dDevice.QueryInterfaceOrNull<InfoQueue>();
```

Comment:
Doesn't this need to be `Disposed` on device destruction?  Will it otherwise keep the graphics device from being release?